.rubocop.yml

```diff
@@ -9,7 +9,7 @@ RSpec/FilePath:
   Enabled: false
 
 RSpec/NestedGroups:
-  Max: 4
+  Max: 5
```

Comment:
😆 Classic solution to a rules violation, just increase what is allowed (not saying it's wrong per se, but maybe we should talk about it before we change the rules in a repo?).